themes/10up-theme/tailwind.config.js

```diff
@@ -45,6 +46,7 @@ module.exports = {
 	],
 	safelist: [
 		{
+			...purgecssWordpress.safelist,
 			pattern: /(bg|text)-(primary|secondary|tertiary)/,
 		},
 	],
```

Comment:
@dainemawer @iansvo Are we sure this is in the right spot? https://tailwindcss.com/docs/content-configuration#using-regular-expressions Based on the docs, I get the feeling it is more supposed to be like:
```suggestion
	safelist: [
		...purgecssWordpress.safelist,
		{
			pattern: /(bg|text)-(primary|secondary|tertiary)/,
		},
	],
```
I could be wrong though and it may not matter. FWIW, in my testing it appeared to work the same. Just want to make sure we're following the guidance.

Reply:
Yeah that is actually how it should be. The object there is only necessary because we're using a pattern. I believe this safelist here is just an array of strings, so I would say lets change it as Jon suggests.

Reply:
Great catch! Thanks @iansvo !